python/observability-online-eval/openai/computer-use/agent.py

```
@@ -0,0 +1,158 @@
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)